cmd/provendb-verify/cli.go

```diff
@@ -253,6 +257,13 @@ func handleCLI(c *cli.Context) int {
 			})
 		}
 
+		if ignoredCollections := c.String("ignoredCollections"); ignoredCollections != "" {
```

Comment:
here should use `c.StringSlice("ignoredCollections")`

Reply:
Fixed.